OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -170,6 +170,13 @@ public void InflictDamage(Actor self, Actor attacker, Damage damage, bool ignore
 					.Concat(damageModifiersPlayer)
 					.Select(t => t.GetDamageModifier(attacker, damage));
 
+				if (attacker.Owner.Handicap > 0)
+				{
+					var list = modifiers.ToList();
+					list.Add(attacker.Owner.Handicap);
```

Comment:
This looks a bit weird. Maybe do

```suggestion
					modifiers.Concat(new[] { attacker.Owner.Handicap });
```

instead?

Reply:
When I have time I will give some broader advice on how to integrate the feature following our standard mechanisms. Focusing on specifics like this isn't too useful until we have the overall approach sorted.

Reply:
Yeah, this needs to go away anyway.